cms/standard_pages/migrations/0002_rename_body_informationpage_content.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.2 on 2024-11-07 17:55
```

Comment:
I had everything in one migration file so far. It can be in one migration file that's no poblem, I left it for teaching purposes for Neha. Was going to make into one today before merging. Will update this once I go through everything with Neha =)

Reply:
Will make sure to push 1 migration file in a PR as I could of made into 1 migration file and pushed it up. Then locally for teaching purposes I could of done something else. Will keep this in mind for the future.